File: Server/src/main/java/com/example/Server/dto/response/productVariant/ProductVariantResponse.java
package com.example.Server.dto.response.productVariant;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ProductVariantResponse {

    private String id;
    private Integer quantity;
    private String img;
    private String productId;
    private String colorCode;
    private String sizeId;
}

File: Client/src/type/ProductVariant/ProductVariantResponse.ts
import type { PageResponse } from "../../api/BaseApi/baseApi";

export interface ProductVariantResponse {
  id: string;
  quantity: number;
  img: string;
  productId: string;
  colorCode: string;
  sizeId: string;
}

export type ProductVariantResponsePageResponse = PageResponse<ProductVariantResponse>;